org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -0,0 +1,54 @@
+# --- Signatures ---
```

Comment:
Do you want to merge this test template file?

Reply:
sure

Reply:
Why do we need it? I feel like we should keep our code repository clean and concise.